handler/src/main/java/io/netty/handler/ssl/OpenSslEngine.java

```diff
@@ -531,6 +553,27 @@ private void checkPendingHandshakeException() throws SSLHandshakeException {
         }
     }
 
+    /**
+     * Log the error, shutdown the engine and throw an exception.
+     */
+    private void shutdownWithError(String operations) throws SSLException {
+        String err = SSL.getLastError();
+        shutdownWithError(operations, err);
+    }
+
+    private void shutdownWithError(String operations, String err) throws SSLException {
+        if (logger.isDebugEnabled()) {
+            logger.debug(operations + " failed: OpenSSL error: '" + err + '\'');
```

Comment:
Too many colons, and I guess there's no reason to wrap `err` with single quotes. How about `" failed due to an OpenSSL error: " + err`?